Assets/Editor/GlitchEscape/CustomShaderGraph.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
directory meta files shouldn't be imported - see [https://github.com/glitch-escape/glitch-escape/pull/104](https://github.com/glitch-escape/glitch-escape/pull/104)

delete this file, and if it's not in the gitignore for some reason uhh let me know?